src/test/ts/part3/Part3Ex2Test.ts

```diff
@@ -112,5 +114,6 @@ describe('Part3Ex3Test', () => {
     ].join('\n'));
 
     // TODO: Write an assertion to test your changes (hint: TinyAssertions)
+    // Isn't the assertion above already doing that ?
```

Comment:
That seems to be a fairly recent change. We should note this and update the original maybe remove the assertContent call.